meta-lmp-base/recipes-tpm/swtpm/swtpm_%.bbappend

```diff
@@ -0,0 +1,12 @@
+# BUG STATEMENT:
+# Upstream recipe is breaking due to a parsing error when static IDs are enabled:
+# ERROR: build/conf/../../layers/meta-security/meta-tpm/recipes-tpm/swtpm/swtpm_0.2.0.bb: argument -d/--home-dir: expected one argument
+# ERROR: build/conf/../../layers/meta-security/meta-tpm/recipes-tpm/swtpm/swtpm_0.2.0.bb: swtpm: Unable to parse arguments for USERADD_PARAM_swtpm '--system -g tss --home-dir      --no-create-home  --shell /bin/false swtpm':
+# ERROR: Failed to parse recipe: build/conf/../../layers/meta-security/meta-tpm/recipes-tpm/swtpm/swtpm_0.2.0.bb
+#
+# FIX:
+# Drop --home-dir parameter
+#
+# TODO: Submit upstream
+USERADD_PARAM_${PN}_lmp = "--system -g ${TSS_GROUP} \
+    --no-create-home  --shell /bin/false ${BPN}"
```

Comment:
Do we need no-create-home when home dir is not specified?

Reply:
Hmm.. apparently you do.  From `man useradd`:
```
The default is to append the LOGIN name to BASE_DIR and use that as the login directory name. The directory HOME_DIR does not have to exist but will not be created if it is missing.
```